src/components/hassio-card-content.html

```diff
@@ -0,0 +1,76 @@
+<link rel='import' href='../../bower_components/polymer/polymer-element.html'>
+<link rel='import' href='../../bower_components/iron-icon/iron-icon.html'>
+
+<link rel='import' href='./ha-relative-time.html'>
+<link rel='import' href='../util/hass-util.html'>
+
+<dom-module id='hassio-card-content'>
+  <template>
+    <style>
+      iron-icon {
+        color: var(--secondary-text-color);
+        margin-right: 16px;
+        margin-top: 16px;
+        float: left;
+      }
+      iron-icon.update {
+        color: var(--paper-orange-400);
+      }
+      iron-icon.running,
+      iron-icon.installed {
+        color: var(--paper-green-400);
+      }
+      iron-icon.primary {
+        color: var(--primary-color);
+      }
+      .title {
+        white-space: nowrap;
+        text-overflow: ellipsis;
+        overflow: hidden;
+      }
+      .description {
+        color: var(--secondary-text-color);
+        overflow: hidden;
+        position: relative; 
+        height: 2.4em;
+        line-height: 1.2em;
+      }
+    </style>
+    <iron-icon icon='[[icon]]' class$='[[iconClass]]' title='[[iconTitle]]'></iron-icon>
+    <div>
+      <div class='title'>[[title]]</div>
+      <div class='description'>
+        <template is='dom-if' if='[[!datetime]]'>
+          [[description]]
+        </template>
+        <template is='dom-if' if='[[datetime]]'>
+          <ha-relative-time datetime='[[description]]'></ha-relative-time>
```

Comment:
This is weird. Don't change the meaning of the value of a variable based on a boolean. Why not just have datetime be a string and pass that to `ha-relative-time` when available?